packages/editor/src/plugins/fill-in-the-blanks-exercise/renderer.tsx

```diff
@@ -138,20 +137,18 @@ export function FillInTheBlanksRenderer(props: FillInTheBlanksRendererProps) {
       {/* Only show "Stimmt's?" during render/preview*/}
       {!isEditing && (
         <div className="mt-2 flex">
-          <button
-            className={cn(
-              'serlo-button-blue mr-3 h-8',
-              allBlanksHaveText && blanks.length > 0
-                ? ''
-                : 'pointer-events-none opacity-0'
-            )}
-            onClick={() => {
-              checkAnswers()
-              setShowFeedback(true)
-            }}
-          >
-            {exStrings.check}
-          </button>
+          {allBlanksHaveText ? (
+            <button
+              data-qa="plugin-exercise-check-answer-button"
+              className="serlo-button-blue mr-3 h-8"
+              onClick={() => {
+                checkAnswers()
+                setShowFeedback(true)
+              }}
+            >
+              {exStrings.check}
+            </button>
+          ) : null}
```

Comment:
Could you please elaborate on this change? I guess an invisible element was a problem for the tests, but why change the condition itself as well (leaving out `blanks.length > 0`)?